src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -1099,6 +1101,10 @@ protected T computeNext() {
         return stream(() -> iterator);
     }
 
+    public static Stream<VariantContext> streamVcf(final File vcf) {
```

Comment:
I don't think Utils is the right place for this method.  I think it should maybe be in VariantContextTestUtils instead.

Reply:
done